pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -75,8 +75,14 @@ def init_meta(self):
 
     def compound_meta(self, headarr, meta_key):
         if meta_key == 'binning':
-            binspatial = headarr[0]['HIERARCH ESO DET WIN1 BINX']
-            binspec = headarr[0]['HIERARCH ESO DET WIN1 BINY']
+            try:
```

Comment:
Let's avoid using `try/except` unless we absolutely have to.

Use an if statement here, (i.e. query for the key).

Reply:
Good point!
I have updated it with:
if 'HIERARCH ESO DET WIN1 BINX' in headarr[0]:
and
if 'HIERARCH ESO DET WIN1 BINY' in headarr[0]:

Reply:
@profxj is this good now?

Reply:
yes, thanks